CHANGELOG.md

```diff
@@ -71,6 +71,7 @@ Here is an overview of all new **experimental** features:
 
 - **GCP Scalers**: Added custom time horizon in GCP scalers ([#5778](https://github.com/kedacore/keda/issues/5778))
 - **Kafka**: Fix logic to scale to zero on invalid offset even with earliest offsetResetPolicy ([#5689](https://github.com/kedacore/keda/issues/5689))
+- *Cassandra Scaler*: Add TLS support for cassandra scaler ([#5802](https://github.com/kedacore/keda/issues/5802))
```

Comment:
```suggestion
- **Cassandra Scaler**: Add TLS support for cassandra scaler ([#5802](https://github.com/kedacore/keda/issues/5802))
```

Reply:
@JorTurFer I did not really understand, the part when you say 
We have to remove the credentials during the scaler Close()

I could not find the ame in kafkascaler(). Could you please help me with m,ore details.

Reply:
Currently, we generate the file with the certificate during the scaler generation, but if I remove the ScaledObject, the temp file with the certificate isn't removed. 

In Kafka scaler, we remove the certificate/secret from the filesystem during the `Close()` function of the scaler:
https://github.com/kedacore/keda/blob/7bcfb69e23101de9644ceae2e0ecb753695309e9/pkg/scalers/kafka_scaler.go#L839-L857

Reply:
Added steps to remove temp file